.github/workflows/e2e.yaml

```
@@ -64,9 +66,14 @@ on:
         required: true
       workflow_trigger:
         type: string
+      cluster_name:
+        type: string
+        description: If this cluster_name is an empty string or isn't specified, a new cluster will be created. A specified cluster_name, will run tests on an existing cluster
```

What do you think about this?
```suggestion
        description: If cluster_name is empty, a new cluster will be created. Otherwise, tests will run on an existing cluster
```